src/lib/core/fiber.h

```diff
@@ -1121,6 +1121,27 @@ fiber_c_invoke(fiber_func f, va_list ap)
 	return f(ap);
 }
 
+/**
+ * Whether debug mode is enabled on not.
+ *
+ * Currenly in debug mode we collect fiber gc allocation backtrace and print
+ * it if leak is found. Otherwise we only print that leak is found.
+ */
+extern bool fiber_debug_enabled;
+
+/**
+ * If false then we don't abort in debug build if fiber gc leak is found.
+ * Default is true. Used in tests.
+ */
+extern bool fiber_abort_gc_leak;
```

Comment:
The fact that this flag only affects debug builds looks confusing. I think it should affect release builds as well - we just need to initialize it to `false` then, similarly to `fiber_debug_enabled`.

Reply:
Fixed.